contracts/TopShotMarketV3.cdc

```diff
@@ -138,6 +139,9 @@ pub contract TopShotMarketV3 {
             pre {
                 self.ownerCollection.borrow()!.borrowMoment(id: tokenID) != nil:
                     "Moment does not exist in the owner's collection"
+
```

Comment:
I think you will need to add this to the [MarketTopShotOldVersion.cdc](https://github.com/dapperlabs/nba-smart-contracts/blob/master/contracts/MarketTopShotOldVersion.cdc) as well? since NBA has to support both contracts (If a moment was listed with the old version then the V3 checks it and processes either the delisting or purchasing with the old contract)

Reply:
Ohh yes, cool, no update needed then 👍